pom.xml

```diff
@@ -143,7 +151,6 @@
         <groupId>org.apache.camel</groupId>
         <artifactId>camel-test-blueprint</artifactId>
         <version>${camel.version}</version>
-        <scope>test</scope>
```

Comment:
If this dependency will always be `test` scope, may as well leave it defined here.